### DISCUSSION TITLE
Rendering a contract award notice using the visualisation API doesn't render the result section

### DISCUSSION THREAD

**Discussion by MichaelAdam-AKA-Mike** [OP-TED/eForms-SDK#828] [Q&A] [answered]
Hi,

we use the visualisation API to render the html for the eForms xml files sent us us from different eTendering platforms.

When  sending a contract award notice the result section <efac:NoticeResult> is not rendered in html.

Here is the beginning of the xml-document:

&lt;?xml version="1.0" encoding="UTF-8"?&gt;
&lt;ContractAwardNotice xmlns="urn:oasis:names:specification:ubl:schema:xsd:ContractAwardNotice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2" xmlns:efac="http://data.europa.eu/p27/eforms-ubl-extension-aggregate-components/1" xmlns:efbc="http://data.europa.eu/p27/eforms-ubl-extension-basic-components/1" xmlns:efext="http://data.europa.eu/p27/eforms-ubl-extensions/1" xmlns:ext="urn:oasis:names:specification:ubl:schema:xsd:CommonExtensionComponents-2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"&gt;
&lt;ext:UBLExtensions&gt;
&lt;ext:UBLExtension&gt;
&lt;ext:ExtensionContent&gt;
&lt;efext:EformsExtension&gt;
&lt;efac:NoticeResult&gt;
&lt;efac:LotResult&gt;
&lt;cbc:ID schemeName="result"&gt;RES-0001&lt;/cbc:ID&gt;
&lt;cbc:TenderResultCode listName="winner-selection-status"&gt;clos-nw&lt;/cbc:TenderResultCode&gt;
&lt;efac:DecisionReason&gt;
&lt;efbc:DecisionReasonCode listName="non-award-justification"&gt;no-rece&lt;/efbc:DecisionReasonCode&gt;
&lt;/efac:DecisionReason&gt;
&lt;efac:ReceivedSubmissionsStatistics&gt;
&lt;efbc:StatisticsCode listName="received-submission-type"&gt;tenders&lt;/efbc:StatisticsCode&gt;
&lt;efbc:StatisticsNumeric&gt;0&lt;/efbc:StatisticsNumeric&gt;
&lt;/efac:ReceivedSubmissionsStatistics&gt;
&lt;efac:ReceivedSubmissionsStatistics&gt;
&lt;efbc:StatisticsCode listName="received-submission-type"&gt;t-esubm&lt;/efbc:StatisticsCode&gt;
&lt;efbc:StatisticsNumeric&gt;0&lt;/efbc:StatisticsNumeric&gt;
&lt;/efac:ReceivedSubmissionsStatistics&gt;
&lt;efac:ReceivedSubmissionsStatistics&gt;
&lt;efbc:StatisticsCode listName="received-submission-type"&gt;t-sme&lt;/efbc:StatisticsCode&gt;
&lt;efbc:StatisticsNumeric&gt;0&lt;/efbc:StatisticsNumeric&gt;
&lt;/efac:ReceivedSubmissionsStatistics&gt;
&lt;efac:TenderLot&gt;
&lt;cbc:ID schemeName="Lot"&gt;LOT-0000&lt;/cbc:ID&gt;
&lt;/efac:TenderLot&gt;
&lt;/efac:LotResult&gt;
&lt;/efac:NoticeResult&gt;
&lt;efac:NoticeSubType&gt;
&lt;cbc:SubTypeCode listName="notice-subtype"&gt;29&lt;/cbc:SubTypeCode&gt;
&lt;/efac:NoticeSubType&gt;

But the result section in html, that can be seen on TED website:

6. Ergebnisse
6.1.
Ergebnis, Los-– Kennung: LOT-0000
Es wurde kein Wettbewerbsgewinner ermittelt, und der Wettbewerb ist abgeschlossen.
Grund, warum kein Gewinner ausgewählt wurde: Es sind keine Angebote, Teilnahmeanträge oder Projekte eingegangen
6.1.4.
Statistische Informationen:
Eingegangene Angebote oder Teilnahmeanträge:
Art der eingegangenen Einreichungen: Angebote
Anzahl der eingegangenen Angebote oder Teilnahmeanträge: 0
Art der eingegangenen Einreichungen: Angebote auf elektronischem Wege eingereicht
Anzahl der eingegangenen Angebote oder Teilnahmeanträge: 0
Art der eingegangenen Einreichungen: Angebote von Kleinst-, kleinen oder mittleren Unternehmen
Anzahl der eingegangenen Angebote oder Teilnahmeanträge: 0

is missing the response from the visualisation API.

Any ideas why this happens? Are there still errors in the API rendering eforms-sdk-1.7?

Kind regards

Michael

**Comment by YvesJo** [comment, marked as answer]
Hi,
For this specific case, please get in touch directly with the helpdesk ted@publications.europa.eu and attach the whole XML file.
Thanks

**Reply by MichaelAdam-AKA-Mike** [reply]
Hi,

​I did it like that.

Thanks
